typings.d.ts

```diff
@@ -0,0 +1,54 @@
+declare module 'adm-zip';
```

Comment:
Can you guys explain the use case of this file i.e. `typings.d.ts`?

Reply:
We define common types we are likely to use in many places here, like `page_params` etc., so like a global types file. Also, since there are not types of some modules we need to use this file also.